components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.user.create/public/js/bottomJs.js

```diff
@@ -254,7 +254,7 @@ $(document).ready(function () {
                     if (data.errorMessage) {
                         $(errorMsg).text("Selected user store prompted an error : " + data.errorMessage);
                         $(errorMsgWrapper).removeClass("hidden");
-                    } else if (data["status"] == 201) {
+                    } else if (data["statusCode"] == 201) {
```

Comment:
As this is a js file, we don't have built in status code constants. So we need to add constants object in to the js to do that. But it will not gave any additional advantage, but reducing the performance slightly.